assests/css/styles.css

```diff
@@ -33,15 +32,23 @@ nav {
   height: 20px;
   position: relative;
   color: #6070ff;
-  top: 58px;
+  top: 70px;
+}
+
+.contaner {
+  width: 375px;
+  background: url(/assests/images/header-background-mobile.png);
 }
 
-section {
+.section-1 {
+  width: 375px;
```

Comment:
Kindly, I suggest you not to put a static width, it is better to use a percentage so that your site adapts to any size of mobile device